koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -270,12 +270,14 @@ def get_expected_nvr(self):
     def format_label(self, label_id):
         """Formats string with user-facing LABEL name and its alternatives"""
 
-        assert label_id in LABEL_NAME_MAP
+        if not label_id in LABEL_NAME_MAP:
+            dockerfile_parse.parser.logger.error('"%s" does not contain "%s"',
```

Comment:
Raise error

Reply:
Just for my curiosity, why `dockerfile_parse.parser.logger.error` ?

Reply:
There's no module-level logger, and `LabelsWrapper()` doesn't have one

Reply:
so you are going to log issue in koji-c as issue in dockerfile-parse ?

Reply:
Moot point as we'll `raise` rarther than simply log